WcaOnRails/lib/database_dumper.rb

```diff
@@ -521,6 +521,8 @@ def self.actions_to_column_sanitizers(columns_by_action)
     }.freeze,
     "vote_options" => :skip_all_rows,
     "votes" => :skip_all_rows,
+    "people_pairs_with_competition" => :skip_all_rows,
+    "linkings" => :skip_all_rows,
```

Comment:
Ah sure, I guess my first thought was that auxiliary data doesn't necessarily need to land here, but it certainly would be a good thing!